src/main/java/com/wavefront/helpers/TraceTypePatternSanitizer.java

```diff
@@ -0,0 +1,59 @@
+package com.wavefront.helpers;
+
+import com.fasterxml.jackson.databind.util.StdConverter;
+import com.wavefront.SpanSender;
+import com.wavefront.TraceTypePattern;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class TraceTypePatternSanitizer extends StdConverter<TraceTypePattern, TraceTypePattern> {
+  private static final Logger LOGGER = Logger.getLogger(SpanSender.class.getCanonicalName());
+  private final static List<TraceTypePattern.TagVariation> DEFAULT_MANDATORY_TAGS =
+      new ArrayList<>(
+          List.of(
+              new TraceTypePattern.TagVariation("application",
+                  new ArrayList<>(Arrays.asList("Application_1", "Application_2"))),
```

Comment:
I will prefer to have only one application as default, as more then one application set is preserved for advanced mode.

Reply:
I also removed some redundant new-s